builder/docker-rules.mk

```diff
@@ -24,6 +24,8 @@ IMAGE_NAME ?=           $(NAME)
 IMAGE_BOOTSCRIPT ?=     stable
 S3_FULL_URL ?=          $(S3_URL)/$(FULL_NAME).tar
 ADDITIONAL_ASSETS ?=
+USE_INTERNAL_HTTPD ?=	$(shell mkdir -p /tmp/build; marker=$$(date); echo $$marker > /tmp/build/httpd_check; test "$$(curl --silent --noproxy '*' http://127.0.0.1/httpd_check)" = "$$marker" ; echo $$?; rm /tmp/build/httpd_check)
```

Comment:
I prefer to move this check and the following `ifeq USE_INTERNAL_HTTPD` in the `image_on_local` rule to avoid doing this check each time we run the `make ...` command